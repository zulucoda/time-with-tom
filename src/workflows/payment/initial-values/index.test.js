import initialValues from '.';

describe('Workflows - User - Initial Values - Unit Test', () => {
  test('should return initial values', () => {
    const act = initialValues;
    expect(act).toMatchSnapshot();
  });
});
