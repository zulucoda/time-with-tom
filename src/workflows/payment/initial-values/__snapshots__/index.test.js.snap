// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Workflows - User - Initial Values - Unit Test should return initial values 1`] = `
Object {
  "ccCvc": 0,
  "ccExpiryMonth": 0,
  "ccExpiryYear": 0,
  "ccName": "",
  "ccNumber": "",
}
`;
